packages/experimental/Expander/src/Expander.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { expanderName, ExpanderType, ExpanderProps, ExpanderViewProps, ExpanderChangeEvent } from './Expander.types';
+import { compose, mergeProps, withSlots, UseSlots, buildProps } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const ExpanderComponent = ensureNativeComponent('MSFExpanderView');
+
+function delay(ms: number) {
+  return new Promise( resolve => setTimeout(resolve, ms) );
+}
+
+export const Expander = compose<ExpanderType>({
+  displayName: expanderName,
+  tokens: [{}, expanderName],
+  slotProps: {
+    root: buildProps(
+      (tokens, theme) => ({
+        ...tokens,
+      }),
+      [],
+    ),
+  },
+  slots: { root: ExpanderComponent },
+  render: (userProps: ExpanderProps, useSlots: UseSlots<ExpanderType>) => {
+    const Root = useSlots(userProps).root;
+    const [expandedState, setExpandedState] = React.useState(userProps.expanded);
+    return (rest: ExpanderViewProps, ...children: React.ReactNode[]) => <Root
+      style={{
```

Comment:
@Saadnajmi Adding style and onChange here causes the pipeline to fail because they aren't in tokens/props. I don't want to expose these to users, so is there somewhere else they should be added to resolve these errors? If I remember correctly, utilizing buildUseStyling states could be an alternative to this?

Reply:
After looking through your code, I think I understand your problem. You have props you need to pass to your Root Slot. These props don't depend on tokens, so you can't really make them in slotProps, which is where I would have suggested doing it to not expose them to the user. I haven't personally used the states part of compose yet, so I'll have to do some research of my own to investigate what a proper solution is.

Reply:
Would making collapsedHeight and expandedHeight tokensThatAreAlsoProps be good design and solve the issue? I made them props since each instance of the Expander could have unique content height and therefore unique expanded and collapsed heights.

Reply:
That's definitely something we did in ActivityIndicator and something that could work. I think it would be a valid design decision. It's a part of styling that we also want to be a customizable property of each control.